src/components/AvatarGroup/AvatarGroupCounterTooltipContent.jsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import Clickable from "../Clickable/Clickable";
+import Flex from "../Flex/Flex";
+import Avatar from "../Avatar/Avatar";
+import cx from "classnames";
+import PropTypes from "prop-types";
+import styles from "./AvatarGroupCounterTooltipContent.module.scss";
+
+const AvatarGroupCounterTooltipContent = ({ avatars, type, className }) => {
+  const getTooltipContent = avatarProps => {
+    return avatarProps?.tooltipProps?.content || avatarProps?.ariaLabel;
+  };
+
+  avatars = avatars.map(avatar => ({ value: avatar.props }));
+  const displayAsGrid = !avatars.some(item => getTooltipContent(item.value));
+
+  const avatarRenderer = (item, index, style) => {
+    const avatarProps = item.value;
```

Comment:
why another avatar renderer? there is one in the avatar component. let's talk about it?

Reply:
also this is a problematic pattern

Reply:
It renders not avatar only avatar itself, but kinda avatarRowItem
Let's talk